marketplace_builder/custom_model_types/car.yml

```diff
@@ -0,0 +1,5 @@
+name: Car
+custom_attributes:
```

Comment:
you can use `properties:`

Reply:
you can use suggest mode to make CR process faster and easier @Slashek  :)

```suggestion
properties:
```